ACadSharp/IO/DXF/DxfStreamReader/DxfObjectsSectionReader.cs

```diff
@@ -59,8 +59,8 @@ private CadTemplate readObject()
 					return this.readObjectCodes<Layout>(new CadLayoutTemplate(), readLayout);
 				case DxfFileToken.ObjectDictionaryVar:
 					return this.readObjectCodes<DictionaryVariable>(new CadTemplate<DictionaryVariable>(new DictionaryVariable()), this.readObjectSubclassMap);
-				//case DxfFileToken.ObjectSortEntsTable:
-				//return this.readSortentsTable();
+				case DxfFileToken.ObjectSortEntsTable:
+					return this.readSortentsTable();
```

Comment:
currently there is an error with the `readSortentsTable()` don't uncomment the method.